grails-app/services/au/org/ala/specieslist/QueryService.groovy

```diff
@@ -606,6 +607,11 @@ class QueryService {
         }
     }
 
+    def validMatchedSpeciesField(String field) {
+        return new String[] {"taxonConceptID", "scientificName", "scientificNameAuthorship", "vernacularName", "kingdom",
+                "phylum", "taxonClass", "taxonOrder", "taxonRank", "family", "genus"}.contains(field) ? field : "scientificName";
```

Comment:
Should we make the match case-insensitive

Reply:
The database column names are case senstive. Case sensitive fields are consistent with other applications. Case sensitive fields are consistent with this application for kvp and regular field:value filtering.